_topic_maps/_topic_map.yml

```diff
@@ -531,8 +534,8 @@ Topics:
   File: configuring-alert-notifications
 - Name: Converting a connected cluster to a disconnected cluster
   File: connected-to-disconnected
-- Name: Cluster capabilities
-  File: cluster-capabilities
+- Name: Enabling cluster capabilities
+  File: enabling-cluster-capabilities
```

Comment:
Should there be a `Distros` filter here too?

Reply:
I'm not sure - I think either both caps pages should have a filter or neither should. I will look into this

Reply:
I've added the distro filter to this entry.